src/network/C4Network2Client.cpp

```diff
@@ -231,6 +231,8 @@ void C4Network2Client::AddLocalAddrs(int16_t iPortTCP, int16_t iPortUDP)
 					addr.SetPort(iPortUDP);
 					AddAddr(C4Network2Address(addr, P_UDP), false);
 				}
+				if (addr.GetScopeId())
```

Comment:
Can someone on Windows verify that this works correctly? E.g. by commenting out line 325 of C4Network2.cpp (above) and testing a local connection (should work even without IPv6 internet connectivity).